starlette/middleware/gzip.py

```diff
@@ -6,29 +6,36 @@
 
 
 class GZipMiddleware:
-    def __init__(self, app: ASGIApp, minimum_size: int = 500) -> None:
+    def __init__(
+        self, app: ASGIApp, minimum_size: int = 500, compresslevel: int = 1
```

Comment:
You should keep it at 9, which is the default value for [GzipFile](https://docs.python.org/3/library/gzip.html#gzip.GzipFile), or this is going to break a lot of existing code.

Reply:
I agree. Ready